### ISSUE TITLE
Bug with geluPlugin

### ISSUE THREAD

**Issue by humuyan** [NVIDIA/TensorRT#1615]
## Description

When I was trying to run an onnx model with `geluPlugin`, an error occurred that the plugin creator could not find attribute `type_id`. However, I have already put `type_id` in the attribute of `CustomGeluPluginDynamic` node.

```
[11/14/2021-15:55:41] [I] [TRT] Searching for plugin: CustomGeluPluginDynamic, plugin_version: 1, plugin_namespace:
[11/14/2021-15:55:41] [E] [TRT] GeluPluginDynamicCreator: invalid typeId -1
[11/14/2021-15:55:41] [E] [TRT] parsers/onnx/ModelImporter.cpp:772: While parsing node number 0 [CustomGeluPluginDynamic -> "output"]:
[11/14/2021-15:55:41] [E] [TRT] parsers/onnx/ModelImporter.cpp:773: --- Begin node ---
[11/14/2021-15:55:41] [E] [TRT] parsers/onnx/ModelImporter.cpp:774: input: "input"
output: "output"
op_type: "CustomGeluPluginDynamic"
attribute {
  name: "type_id"
  i: 0
  type: INT
}
```

It seems that the plugin attributes of `geluPlugin` is not initialized. In `plugin/geluPlugin/geluPlugin.cpp` line 249:

```C++
GeluPluginDynamicCreator::GeluPluginDynamicCreator()
{
    // Fill PluginFieldCollection with PluginField arguments metadata
    mFC.nbFields = mPluginAttributes.size();
    mFC.fields = mPluginAttributes.data();
}
```

After I filled the PluginField arguments metadata:

```C++
GeluPluginDynamicCreator::GeluPluginDynamicCreator()
{
    mPluginAttributes.emplace_back(PluginField("type_id", nullptr, PluginFieldType::kINT32, 1));
    mPluginAttributes.emplace_back(PluginField("bias", nullptr, PluginFieldType::kFLOAT32, 1));

    mFC.nbFields = mPluginAttributes.size();
    mFC.fields = mPluginAttributes.data();
}
```

`trtexec` could execute successfully.

## Environment

**TensorRT Version**: 8.2
**NVIDIA GPU**: Tesla A100
**NVIDIA Driver Version**: 460.73.01
**CUDA Version**: 11.2
**CUDNN Version**: 8.0.4.30-11.1-linux-x64
**Operating System**: Debian 11

**Comment by nvpohanh** [comment]
@ttyio @rajeevsrao Has this been fixed in TRT 8.4?

**Comment by rajeevsrao** [comment]
> @ttyio @rajeevsrao Has this been fixed in TRT 8.4?

Yes, this has been fixed in TensorRT OSS 8.4.
